configure.ac

```diff
@@ -35,6 +36,9 @@ AC_PROG_LIBTOOL
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
 m4_ifdef([AC_PROG_CC_C99],  [AC_PROG_CC_C99])
 
+# check for gtk-doc
+m4_ifdef([GTK_DOC_CHECK], [GTK_DOC_CHECK([1.14],[--flavour no-tmpl])],[AM_CONDITIONAL([ENABLE_GTK_DOC], false)])
```

Comment:
> Icing on the cake if you follow the coding style of the m4_define_default() macro usage right below, instead of the bad copy/pasta job.

I'm not really into the coding style of these files, I just learned half a day ago, what these do. And it was my first time looking into them instead of just using 'em.

Also gtk-doc is not easy to configure / use:
> GTK-Doc wasn't originally intended to be a general-purpose documentation tool, so it can be a bit awkward to setup and use